### DISCUSSION TITLE
Can I skip "Chapter 2 – Neural Network Classification" and go directly to "Chapter 3 – Computer Vision"

### DISCUSSION THREAD

**Discussion by NovaOptio** [mrdbourke/pytorch-deep-learning#804] [General]
Hi,

Im mostly interested in computer vision for a drone project im working on. I did chapter 1 and im now wondering if it would be possible to skip chapter 2 on classification and directly go to chapter 3 on computer vision. 

Thanks in advance

**Comment by skvarre** [comment]
Chapter 3 builds upon Chapter 2 in many ways. For example, the model(s) in Chapter 3 covers a classification problem. However, I think it is possible, though you might need to look up a few things by yourself. I think Daniel is pretty clear when he is talking about something he has covered before, so whenever he does, a short Google search of the concept in question might be all you need.
